src/Smp.Web/Controllers/PostsController.cs

```diff
@@ -1,43 +1,41 @@
-using System;
```

Comment:
What happened here?

Reply:
I think it's one of those `^M` issues, aka the difference in how macOS and Windows mark new line/line feed. I remember seeing it in a diff.